tests/cli/test_deploy_contract_and_verify_helper_fn.py

```diff
@@ -11,72 +11,100 @@
 from populus.utils.testing import load_contract_fixture
 
 
-@load_contract_fixture('Math.sol')
-def test_deploying_contract_with_successful_deploy(project):
+
+def deploy_contract(project,name):
+    
```

Comment:
Same here.  These changes should be reverted as all testing of import remappings should be isolated to the compilation tests.